cmd/collector/app/builder/builder_flags.go

```diff
@@ -27,14 +27,14 @@ import (
 const (
 	collectorQueueSize            = "collector.queue-size"
 	collectorNumWorkers           = "collector.num-workers"
-	collectorPort                 = "collector.port"
-	collectorHTTPPort             = "collector.http-port"
-	collectorGRPCPort             = "collector.grpc-port"
+	collectorTChanAddr            = "collector.tchan-addr"
+	collectorHTTPAddr             = "collector.http-addr"
+	collectorGRPCAddr             = "collector.grpc-addr"
```

Comment:
I would rather refer to these flags as "host-port", as mentioned in the ticket `--http-server.host-port`, not "addr"